src/NNlib.jl

```diff
@@ -100,4 +89,23 @@ include("impl/depthwiseconv_im2col.jl")
 include("impl/pooling_direct.jl")
 include("deprecations.jl")
 
+function __init__()
+    @require NNPACK_jll="a6bfbf70-4841-5cb9-aa18-3a8ad3c413ee" begin
+      if isdefined(NNPACK_jll, :libnnpack)
```

Comment:
I'd rather drop it altogether in the next breaking release because neither we nor any users appear to be using it (NNPack itself has been [basically unmaintained](https://github.com/Maratyszcza/NNPACK/commits/master) since 2020).